main.go

```diff
@@ -50,10 +50,15 @@ type rootCommand struct {
 }
 
 const (
-	version           = "dev"
 	defaultAPICluster = "nineapis.ch"
 )
 
+var (
+	// this has to be a variable as it will be replaced by CI/CD
+	// (goreleaser)
+	version = "dev"
```

Comment:
I like to use this pattern: https://github.com/thde/truenas-scale-acme/blob/main/main.go#L53

It can still be set by the CI, but falls back to the Go native way if not.

Reply:
Thanks, will have a look!

Reply:
Added it. Could you have another look please?